starter/script.js

```diff
@@ -1 +1,153 @@
+      if (qBeesHoney == 'bees') {
+        alert('In this life, you will float like a butterfly, sting like a bee.  You\'re Muhammed Ali');
+      } else if (qBeesHoney == 'honey') {
+        alert('Your life is so SWEET! You\'re so good at boxing that they call you "Sugar" Ray Leonard.');
+      }
+    } else if (qStyle == 'after') {
+        var qWax = prompt('Mixed martial arts is the fighting sport of the day, and you\'re an MMA fighter. One day, while training in kung fu, you have to decide: wax on or wax off (on/off)');
+        if (qWax == 'on') {
+          alert('Oh man, kung fu, karate, and taekwando don\'t do so well in MMA.  You are an MMA fighter in this life. But it hurts. A lot...');
+        } else if (qWax == 'off') {
+          alert('You\'re a Brazilian jujitsu fighter.  You are at the cutting edge of dislocating bones!')
+        }
+    }
+  }
+  playAgain();
+}
+
+function bball() {
+  var qbball = prompt('Prepare to take flight! In this life, your hang time is crazy! Let\'s decide where to take off from: the east or west coast. (east/west)');
+  if (qbball == 'east') {
+    var goPro = prompt('As a teenager, you are more talented at basketball than most professionals.  Do you join the NBA without going to college? (y/n)');
+    if (goPro == 'y') {
+      var beach = prompt('You are born again as Lebron James, three time NBA basketball champion! Now, decide: will you take your talents to South Beach? (y/n)');
+      if (beach == 'y') {
+        alert('Great decision! Miami sure beats cold Cleveland.  You win three championships and enjoy Miami sun!');
+      } else {
+        alert('Um... New York is a great city, but did you really want to play for the Knicks?!?!?!');
+      }
+    } else if (goPro == 'n') {
+      var stay = prompt('You are born again as Andrew Wiggins.\n' +
+      'You could haven went pro early, but you took your time,enjoyed life.\n' +
+      'But will you stay in Minnesota? (y/n)');
+      if ( stay == 'y') {
+        alert('Great decision.  You stay in Minnesota and win the Rookie of the Year Award!');
+      } else {
+        alert('uh oh! You are now on a team where you get less playing time, and still don\t win!');
+      }
+    }
+  } else if (qbball == 'west') {
+    var apparel = prompt('Welcome to North Carolina! Wait. You thought You said WEST! You did, but NC is where you get good at basketball.  And get stalked by apparel companies seeking endorsements. Do you go with Nike or Under Armor? (nike/ua)');
+    if (apparel == 'nike') {
+      var college = prompt('Nike was a no-brainer.  Your favorite ACC teams had Nike deals! You even have the chance to play for one in college.  You are that good.  So, when you declare, where will you go? Are you willing to join the evil empire? (evil / good)');
+      if (college == 'evil') {
+        alert('Duke? Really? Figures. You become JJ Reddick of the Los Angeles Clippsers.  You are as evil as they come!');
```

Comment:
Woahhhhh, you realize I am a huge Duke fan! Just kidding, but I really like your story!